### ISSUE TITLE
Vagabonds opening accounts doesn't let them create custom vendors

### ISSUE THREAD

**Issue by ModDownloading** [discordia-space/CEV-Eris#7930]
1. Join as vagabond
2. Buy custom vendomat board
3. Create custom vendor
4. Run your card through custom vendor, get rejected because you don't have an account
5. Get an account from guild
6. Make sure your account works at the ATM
7. Run your card through custom vendor again, get rejected again

Vagabonds should be able to create custom vending machines as a way to sell their goods in case the guild is short staffed or won't buy your stuff. Just allowing vagabonds to make custom vendors after they have an account would be nice, but ideally it would be really neat for the custom vendor to have a separate option for people without accounts, like holding onto the money in the vendor itself and only allowing the owner to withdraw it (maybe also people who smash the machine can get the money too, especially considering they now have a probably murderous vagabond after them).

**Comment by SirRichardFrancis** [comment]
Try linking the account number to ID card via First Officer's console / ID modification app, that should do it.

**Comment by DiaFRAME444** [comment]
If it works in the ATM, it should have the account number tied in.

**Comment by SirRichardFrancis** [comment]
Think it worked with account data manually entered in the ATM, not by swiping an ID.

**Comment by ModDownloading** [comment]
Yes, I entered the account info manually in the ATM, the ID just happened to be in there too. The fact that the ATM worked probably had nothing to do with the ID.

SirRichardFrancis is probably right that linking the account to the ID should work, but that still seems like a lot of work for a vagabond to set up a vendomat, having to go to the guild and then the first officer/captain to get everything necessary for a single custom vending machine. If someone could implement a way for people without accounts to make custom vending machines I would be very grateful.

Actually, I know this isn't the place for suggestions, but having a set of prebuilt custom vendors where anyone can sell their wares would be pretty neat. People could reserve vending machines to sell stuff, and the guild gets a cut of the sales for providing the service.

**Comment by DiaFRAME444** [comment]
Probably intended, thinking about it.
Vagabonds are low life, they should be selling things by going to buyers directly or using the open areas by the Guild, not by custom Vendomat.

**Comment by Kurgis** [comment]
Sadly IDs are made with antag gameplay and not pure management in mind, otherwise Command could just remotely alter IDs. The best current in-game solution is a First Officer with a tablet in their backpack, who you can have connect your ID to an account "on the fly."

One solution to this issue may be allowing an ATM to connect an account number to an inserted card if it's logged in, and the card's account number is either null or zero (as Vagabond's are). TGstation has quite the robust economy system, I see more people in-game use it due to its convenience, than in Eris' who some often instead just use it as a roleplay aspect. There should be automative aspects to how money is handled, right now is just an inconvenience to gameplay.
